jbehave-support-core/docs/Ssh.md

```diff
@@ -63,18 +63,17 @@ Note that only verifiers applicable for String make sense in this case - usage o
 
 #### Search in specific log part:
 Ssh steps mentioned above by default fetch the log from whole run of the test story.  
-For using just a part of the log a specified named timestamp can be defined during the run of test story using a special command: (timestamp in example uses the name _NCAKO_2_, but any other name can be used)
-
-```
-Given log timestamp is saved as NCAKO_2
-```
-
-Afterwards the same principle of searching the logs as described above applies, but you use the `since [$startTimeAlias]` step variant:
-```
-Then the following data are present in [TEST] log since [NCAKO_2]:
+Theirs behaviour can be modified by setting timestamps (it's highly recommended to set both timestamps!)
 ```
+Given log read start timestamp is set to now
```

Comment:
The wording seems kinda weird, but cant think of a better one right now 🙂

Reply:
Me neider :(

Reply:
new day, new ideas 🙂  what about something like "Given current time is set as a log start timestamp" or something similar? still not great though

Reply:
I like this idea :)